distribution/docs/src/main/resources/content/_whitelists/catalog-whitelist.adoc

```diff
@@ -6,39 +6,167 @@
 :order: 01
 :summary: ${ddf-catalog} whitelist.
 
-The following packages have been exported by the ${ddf-catalog} application and are approved for use by third parties:
-
-* `ddf.camel.component.catalog`
-* `ddf.catalog`
-* `ddf.catalog.cache`
-* `ddf.catalog.data`
-* `ddf.catalog.data.metacardtype`
-* `ddf.catalog.event`
-* `ddf.catalog.federation`
-* `ddf.catalog.federation.impl`
-* `ddf.catalog.filter`
-* `ddf.catalog.filter.delegate`
```

Comment:
❗️Those are common utility classes so I'm not sure we can or should remove them from the whitelist.

Reply:
I can see adding these to the whitelist but it should be understood that these are implementations and can/will change. Looks like most of the classes in this package have changed in the past 1-2 years

Reply:
I don't think we should have any impls in the whitelist, base classes or no.

Reply:
`ddf.catalog.filter.delegate` was intentionally designed as an abstract class.  I don't think it can be embedded since the GeotoolsFilterBuilder and Adapter also use them across bundle classloaders.